repo-scripts/prune-dts/tests/firestore.output.d.ts

```diff
@@ -341,12 +341,15 @@ export declare interface DocumentChange<T = DocumentData> {
  * The type of a `DocumentChange` may be 'added', 'removed', or 'modified'.
  */
 export declare type DocumentChangeType = 'added' | 'removed' | 'modified';
+
+/** Alias dynamic document field value types to any */
+export declare type DocumentFieldValue = any;
 /**
  * Document data (for use with {@link setDoc}) consists of fields mapped to
  * values.
  */
 export declare interface DocumentData {
```

Comment:
wonder why it is sometimes` interface DocumentData`, and sometimes `type DocumentData`

Reply:
lint check is really not happy about the new type `DocumentFieldValue`, most of the errors below is related to it, saying `cannot find the symbol` 🤔